locale/en.po

```diff
@@ -538,7 +538,7 @@ msgstr "Big Iron Chunk"
 
 #: ../simulation_parameters/microbe_stage/biomes.json:148
 msgid "VOLCANIC_VENT"
-msgstr "Volcanic vent"
+msgstr "Volcanic Vent"
```

Comment:
Do we *need* to change these? I kinda don't think this is necessary. Anyone else have thoughts on this?